src/Diagnostics.DataProviders/DataProviders/MockSupportObserverDataProvider.cs

```diff
@@ -63,10 +64,10 @@ public Task<JObject> GetAppServiceEnvironmentDetails(string hostingEnvironmentNa
 
```

Comment:
Above it appears like you are throwing a null exception when stamp is null, but in the method right above you are explicitly calling it with stamp as null so that will never work right?

Reply:
No actually it was unintentional. I don't think you have to worry over this one though. If anything I should remove it because we are always going to call the overloaded version with stampName and siteName